geomesa-blobstore/geomesa-blobstore-api/src/main/scala/org/locationtech/geomesa/blob/api/BlobstoreServlet.scala

```diff
@@ -0,0 +1,51 @@
+package org.locationtech.geomesa.blob.api
+
+import java.io.{File, FileOutputStream}
+
+import com.google.common.io.Files
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreFactory}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore
+import org.locationtech.geomesa.web.core.GeoMesaScalatraServlet
+import org.scalatra.{NotFound, BadRequest, Ok}
+
+import scala.collection.JavaConversions._
+
+class BlobstoreServlet extends GeoMesaScalatraServlet {
+  override def root: String = "blob"
+
+  var abs: AccumuloBlobStore = null
+
+  post("/ds/:alias") {
+
+    println("In ds registration method")
+
+    val dsParams = datastoreParams
+    val ds = new AccumuloDataStoreFactory().createDataStore(dsParams).asInstanceOf[AccumuloDataStore]
+
+    if (ds == null) {
+      NotFound(reason = "Could not load data store using the provided parameters.")
+    } else {
+      abs = new AccumuloBlobStore(ds)
+      Ok()
```

Comment:
in reference to @anthonyccri 's comment above...if we are creating a new blob store it would be nice to send back a 201 Created HTTP response with a Location header to be more RESTful and ensure that the resource was registered at that ID...